Source/WebCore/platform/audio/AudioUtilities.cpp

```diff
@@ -84,24 +84,21 @@ void applyNoise(float* values, size_t numberOfElementsToProcess, float standardD
     std::mt19937 generator(device());
     std::normal_distribution<float> distribution(1, standardDeviation);
 
-    HashMap<float, float> noiseMultipliers;
-    auto computeNoiseMultiplier = [&](float rawValue) {
-        if (!noiseMultipliers.isValidKey(rawValue))
-            return distribution(generator);
+    static constexpr auto maximumTableSize = 2048;
+    static constexpr auto minimumTableSize = 1024;
 
-        auto result = noiseMultipliers.ensure(rawValue, [&] {
-            return distribution(generator);
-        }).iterator->value;
+    size_t unusedCellCount = weakRandomNumber<uint32_t>() % (maximumTableSize - minimumTableSize);
```

Comment:
We may want a comment to explain that we're using this instead of a HashMap in order to avoid heap allocations on the audio rendering thread.